samples/highcharts/demo/bar-force/demo.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Highcharts plugin for creating individual arrow-head like inverted bars.
+ */
+(function (H) {
+    H.addEvent(
+        H.seriesTypes.column,
+        'afterColumnTranslate',
+        function () {
+            const series = this,
+                options = series.options,
+                topMargin = options.topMargin || 0,
+                bottomMargin = options.bottomMargin || 0,
+                idx = series.index;
+
+            if (options.headSize) {
+
+                series.points.forEach(function (point) {
+                    const shapeArgs = point.shapeArgs,
+                        w = shapeArgs.width,
+                        h = shapeArgs.height,
+                        x = shapeArgs.x,
+                        y = shapeArgs.y,
+                        cutLeft = idx !== 0,
+                        cutRight = point.stackY !== point.y || !cutLeft;
+
+                    var len = options.headSize; // in pixels
+
+                    if (point.y < 0) {
+                        len *= -1;
+                    }
+
+                    // Preserve the box for data labels
+                    point.dlBox = point.shapeArgs;
+
+                    point.shapeType = 'path';
+                    point.shapeArgs = {
+                        d: [
+                            ['M', x, y + topMargin],
+                            [
+                                'L',
+                                x + w / 2,
+                                y + topMargin + (cutRight ? len : 0)
+                            ], // arrow
+                            // top side
+                            ['L', x + w, y + topMargin],
+                            // right side
+                            ['L', x + w, y + h],
+                            // bottom side
+                            [
+                                'L',
+                                x + w / 2,
+                                y + h + bottomMargin + (cutLeft ? len : 0)
+                            ],
+                            ['L', x, y + h + bottomMargin],
+                            // left side
+                            ['L', x, y],
+                            // close
+                            ['Z']
+                        ]
+                    };
+
+                });
+            }
+        }
+    );
+}(Highcharts));
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'bar',
+        height: 170,
+        events: {
+            render: function () {
```

Comment:
I think we can just use stack labels instead of the custom render function: https://jsfiddle.net/highcharts/pzevxsb9/